src/tools/tools.service.ts

```diff
@@ -493,7 +493,7 @@ export async function listTools({
             inputSchema: wikipediaTool.inputSchema,
             isExternal: true,
             userDescription:
-              'Retrieve detailed information from Wikipedia.org on a wide range of topics, including famous individuals, locations, organizations, and historical events. Ideal for obtaining comprehensive overviews or specific details on well-documented subjects. May not be suitable for lesser-known or more recent topics. The information is subject to community edits which can be inaccurate.'
+              'Retrieve detailed information from [Wikipedia.org](https://wikipedia.org) on a wide range of topics, including famous individuals, locations, organizations, and historical events. Ideal for obtaining comprehensive overviews or specific details on well-documented subjects. May not be suitable for lesser-known or more recent topics. The information is subject to community edits which can be inaccurate.'
```

Comment:
We want to open it in a new window, this should work `{:target="_blank"}` , but let me try first.

Reply:
Nevermind, it doesn't work, we'll add it on the UI side.